process/project_milestones.md

```diff
@@ -0,0 +1,64 @@
+* Project can demonstrate its application, practice, and adjustments to its governance documentation as a result of regular project operations
+* Project has a robust and mature security posture which could be achieved by one or more of the following:
+  * Threat model
+  * [Joint review](https://github.com/cncf/tag-security/tree/main/assessments#components-of-the-security-review-package)
+  * [Passing level](https://bestpractices.coreinfrastructure.org/en/criteria/0) of the [OpenSSF Best Practice Badge](https://bestpractices.coreinfrastructure.org/)
+    * Note: Completion and maintenance of **a** OpenSSF Best Practice Badge is [a requirement for graduation](https://github.com/cncf/toc/blob/main/process/graduation_criteria.md#graduation-stage)
+* Project documentation includes getting started guides, operating/administration instruction, security call-outs, and other core elements necessary to ease adoption.
+* Project provides information on performance and scalability for deployment options/configurations, autoscaling, or may provide benchmarks in these areas for comparison or evaluation
+* Project captures use cases to showcase how it can be best used to solve common problems.
+  * These may leverage observations and concerns from adopters
+* Project contribution and community documentation clearly defines expectations when contributing (to include non-code contributions).
+* Project defines a process to “build the bench” of maintainers and project leaders such as a Contribution Ladder which considers succession planning and getting maintainers from multiple organizations
+* Project ensures all sub-projects are clearly defined in their maturity level, whether they ship with the core project or can be used independently but is subject to the same governance processes
+* Project refreshes their roadmap planning with a reasonable cadence
+* Project has engaged with a few Technical Advisory Groups to ensure they are considering best practices across multiple technical and governance facets
+* Project has defined a process for managing the conduct of its community, either establishing its own group to do this, leveraging the existing leadership, or deferring to the CNCF
+* Project maintains a dependency graph so adopters fully understand the complexity and risk of use
+* TBD - add more!
+
+## Graduation
+
+Graduation milestones are guiding points to highlight continuing success areas for highly mature project after they graduate. They are not requirements. Not all Graduated projects will achieve or perform these milestones, however many robust and mature projects will adopt these milestones as "regular business" to maintain the health and growth of the project and its community.
+* Project reviews it's governance process annually for improvements to improve clarity, consistency, and inclusivity
```

Comment:
```suggestion
* Project regularly updates its governance documentation to accurately reflect changes to processes, staffing, roles, and activities
```

We're already giving Incubating projects the full milestone of "complete governance", which means that all we can ask graduated projects to do is to keep that up to date.  No?